cli/connect/lua/eval_func_body.lua

```diff
@@ -25,4 +58,46 @@ for i=2,ret.n do
         ret[i] = box.NULL
     end
 end
+if not need_metainfo then
+    return yaml.encode({unpack(ret, 2, ret.n)})
+end
+
+if ret.n == 2 and type(ret[2]) == 'cdata' and ret[2] ~= nil and ret[2].format then
+    local ret_with_format = {
+        metadata = ret[2]:format(),
+        rows = { ret[2] },
+    }
+    if #ret_with_format.metadata > 0 then
+        ret[2] = ret_with_format
+    end
+end
```

Comment:
What if we have ret[3] and ret[4], or ret[6]? It should be showed as two different tables in the current logic.

Please, add a test for it.

Reply:
Updated. New tests added.